### DISCUSSION TITLE
Hello, what is this?

### DISCUSSION THREAD

**Discussion by pclouds** [shirok/Gauche#764] [General]
I just noticed this new "Discussions" tab. Are we supposed to use this for questions and stuff and reserve "Issues" for actual issues?

**Comment by shirok** [comment]
At this moment the # of issues are not overwhelming and I don't mind keeping everything in the issues tab.  But if you feel that the topic is too abstract or too vague to be an "issue", try this out.

My concern is that this "Discussion" is bound to github, while the repository can move in a long run.  We did migrate from SourceForge.   It is certainly convenient if everything in one place at github, though.  So I'm ambivalent.
